### ISSUE TITLE
Autogen.sh failed on MinGw32

### ISSUE THREAD

**Issue by Behrouz-m** [bagder/libbrotli#27]
I am getting this error:

``` shell
$ ./autogen.sh
./autogen.sh: line 15: /c/Program: No such file or directory
./autogen.sh: line 16: /c/Program: No such file or directory
configure.ac:8: installing './install-sh'
configure.ac:8: installing './missing'
Makefile.am:5: error: Libtool library used but 'LIBTOOL' is undefined
Makefile.am:5:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
Makefile.am:5:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
Makefile.am:5:   If 'LT_INIT' is in 'configure.ac', make sure
Makefile.am:5:   its definition is in aclocal's search path.
Makefile.am: installing './depcomp'
/usr/local/share/automake-1.15/am/depend2.am: error: am__fastdepCC does not appear in AM_CONDITIONAL
/usr/local/share/automake-1.15/am/depend2.am:   The usual way to define 'am__fastdepCC' is to add 'AC_PROG_CC'
/usr/local/share/automake-1.15/am/depend2.am:   to 'configure.ac' and run 'aclocal' and 'autoconf' again
Makefile.am: error: C source seen but 'CC' is undefined
Makefile.am:   The usual way to define 'CC' is to add 'AC_PROG_CC'
Makefile.am:   to 'configure.ac' and run 'autoconf' again.
autoreconf: automake failed with exit status: 1
```

**Comment by bagder** [comment]
That appears as if it has troubles with a space in the path name. Can you try to rerun it if you move it to a path without space?

**Comment by bagder** [comment]
Or perhaps better, run it with `sh -x autogen.sh` and show us what it tried to do there on line 15-16

**Comment by Behrouz-m** [comment]
The problem solved by running `libtoolize`.

**Comment by Behrouz-m** [comment]
path problem was with `git` path.

**Comment by Behrouz-m** [comment]
new problem with make:

``` shell
$ make
make  all-am
make[1]: Entering directory `/c/dev/lib/libbrotli'
make[1]: *** No rule to make target `brotli/dec/streams.c', needed by `brotli/dec/libbrotlidec_la-streams.lo'.  Stop.
make[1]: Leaving directory `/c/dev/lib/libbrotli'
make: *** [all] Error 2
```

**Comment by bagder** [comment]
so did the git command run and do the submodule init and submodule update commands?

**Comment by bagder** [comment]
Ah, the latest brotli code has removed that source file...

**Comment by bagder** [comment]
I think this is now fixed in commit 24a03d284f

**Comment by Behrouz-m** [comment]
It's been built! Thanks a lot.
